uber/config.py

```diff
@@ -195,6 +195,11 @@ def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
     @property
```

Comment:
I'm pretty sure this will raise an exception if you actually try to use it.  (Or if would if you called the ``session.admin_attendee()`` function instead of just returning the method without calling it, which I'm guessing was a mistake.)

One thing that's not intuitive about SQLAlchemy models is that they're either "attached" or "detached".  For example:

```python
    # I added this to the Config class for testing/demonstration purposes:
    @property
    def SOME_ATTENDEE(self):
        with sa.Session() as session:
            return session.query(sa.Attendee).first()

...
>>> c.SOME_ATTENDEE.first_name
Traceback (most recent call last):
...
sqlalchemy.orm.exc.DetachedInstanceError: Instance <Attendee at 0x7f467b6027b8> is not bound to a Session; attribute refresh operation cannot proceed
```

Basically, if you retrieve a model instance from the database and then the session closes, you can no longer access data on that model, on the logic that you no longer have any idea whether the data is still valid.  This is why ``admin_attendee()`` is a method on the ``session`` class instead of on the ``c`` object.

With that being said, there's a valid use case here, which is that maybe we want to just return whatever is in the database at the moment and we don't care if it gets modified 1 millisecond later (because how often does the current admin edit e.g. their own name, anyway?).

In this case, the idiomatic SQLAlchemy approach would be to return the data rather than the model itself, e.g.

```python
    @property
    def CURRENT_ADMIN_NAME(self):
        with sa.Session() as session:
            return session.admin_attendee().full_name
```

or if you wanted to return more of the data, you could use the ``to_dict()`` method, e.g.

```python
    @property
    def CURRENT_ADMIN_NAME(self):
        with sa.Session() as session:
            return session.admin_attendee().to_dict()
```